keyboards/handwired/cyberstar/readme.md

```diff
@@ -0,0 +1,32 @@
+# Cyberstar
+
+This is a 60% with nav claster layout custom PCB. It supports VIA.
+
+* Keyboard Maintainer: [Yiancar](http://yiancar-designs.com/) and on [GitHub](https://github.com/yiancar)
+* Hardware Supported: A custom keyboard with STM32F072CB
+* Hardware Availability: COntact YungBirk#1714 on discord 
+
+## Instructions
+
+### Build
+
+Make example for this keyboard (after setting up your build environment):
+
+    make cyberstar:default
```

Comment:
Actually this is sitting in `keyboards/handwired` do you want to either correct the location or the make example?

Reply:
Thanks boss now fixed!